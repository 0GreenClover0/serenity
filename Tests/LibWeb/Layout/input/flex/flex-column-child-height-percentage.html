<!doctype html><style>
    * {
        outline: 1px solid black !important;
    }
    .flex-item {
        flex-grow: 1;
    }
    .box {
        height: 100%;
    }
    .flex-container {
        display: flex;
        min-height: 100px;
        flex-direction: column;
    }
</style><div class="flex-container"><div class="flex-item"><div class="box">I make music you can dream to.</div><div>My SoundCloud</div></div></div>
