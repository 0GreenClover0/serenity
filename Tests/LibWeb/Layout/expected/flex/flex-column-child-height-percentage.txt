Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x116 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x100 children: not-inline
      Box <div.flex-container> at (8,8) content-size 784x100 flex-container(column) [FFC] children: not-inline
        BlockContainer <div.flex-item> at (8,8) content-size 784x100 flex-item [BFC] children: not-inline
          BlockContainer <div.box> at (8,8) content-size 784x17.46875 children: inline
            line 0 width: 259, height: 17.46875, bottom: 17.46875, baseline: 13.53125
              frag 0 from TextNode start: 0, length: 30, rect: [8,8 259x17.46875]
                "I make music you can dream to."
            TextNode <#text>
          BlockContainer <div> at (8,25.46875) content-size 784x17.46875 children: inline
            line 0 width: 119.03125, height: 17.46875, bottom: 17.46875, baseline: 13.53125
              frag 0 from TextNode start: 0, length: 13, rect: [8,25.46875 119.03125x17.46875]
                "My SoundCloud"
            TextNode <#text>
      BlockContainer <(anonymous)> at (8,108) content-size 784x0 children: inline
        TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x116]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x100]
      PaintableBox (Box<DIV>.flex-container) [8,8 784x100]
        PaintableWithLines (BlockContainer<DIV>.flex-item) [8,8 784x100]
          PaintableWithLines (BlockContainer<DIV>.box) [8,8 784x17.46875]
            TextPaintable (TextNode<#text>)
          PaintableWithLines (BlockContainer<DIV>) [8,25.46875 784x17.46875]
            TextPaintable (TextNode<#text>)
      PaintableWithLines (BlockContainer(anonymous)) [8,108 784x0]